internal/metrics/metrics.go

```diff
@@ -85,6 +97,41 @@ func NewMetrics(registry *prometheus.Registry) *Metrics {
 			[]string{"branch", "revision", "version"},
 		),
 		proxyMetricCache: &RouteMetric{},
+		deprecatedProxyTotalGauge: prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Name: DeprecatedHTTPProxyTotalGauge,
+				Help: "(Deprecated): Total number of HTTPProxies that exist regardless of status.",
```

Comment:
It'd be helpful to include the name of the metric that replaces this one in the help text -- "Use contour_httpproxy instead." at the end of the help text should be sufficient. Same goes for other metrics.

Reply:
yes.. good suggestion. fixed